integration_tests/cli/auth-demo.sh

```diff
@@ -682,4 +650,324 @@ echo "$user adds Model with VID: $vid PID: $pid"
 result=$(echo "test1234" | dcld tx model add-model --vid=$vid --pid=$pid --productName="$productName" --productLabel="Device Description" --commissioningCustomFlow=0 --deviceTypeID=12 --partNumber=12 --from=$user_address --yes 2>&1) || true
 check_response_and_report "$result" "key not found" raw
 
```

Comment:
It would be good to explain here the purpose of the new tests. i.e mentioning that vendor accounts require less quorum of trustees.

Reply:
Done! ✅